plugins/module_utils/client/discovery.py

```diff
@@ -0,0 +1,160 @@
+# Copyright [2017] [Red Hat, Inc.]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
```

Comment:
These two lines are for py2. We can safely drop them now.

Reply:
The sanity tests (for 3.7) fail if they aren't in there. How do I turn those off?

Reply:
I believe it's these two lines:
https://github.com/ansible-collections/vmware.vmware_rest/blob/main/tests/sanity/ignore-2.11.txt#L17-L18